rwt_app_chooser/README.md

```diff
@@ -0,0 +1,111 @@
+rwt_app_chooser
+===============
+
+App chooser client for web browsing device
+
+### Usage
+
+1. (On Robot) Launch app manager
+
+    ```bash
+    roscore &
+    rosparam set robot/type pr2     # set robot type as rosparam
+    rosparam set robot/type pr1012  # set robot name as rosparam
```

Comment:
@furushchev typo? `robot/type` -> `robot/name` ???

Reply:
@k-okada Ah, it's typo. Sorry..